package.json

```diff
@@ -95,6 +95,7 @@
   },
   "dependencies": {
     "babel-runtime": "6.9.2",
-    "vue": "1.0.24"
+    "vue": "1.0.24",
+    "vue-multiselect": "^1.0.1"
```

Comment:
Yes, rawgit enough. It is only a developer example file. 
Please update package.json from `master`. Yesterday I updated dependencies https://github.com/icebob/vue-form-generator/pull/24 and you have conflict with it currently.